packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { string, element, bool } from 'prop-types';
+import styled from 'styled-components';
+import {
+  C_CONSENT_BACKGROUND,
+  C_CONSENT_ACTION,
+  C_CONSENT_CONTENT,
+  C_WHITE,
+} from '@bbc/psammead-styles/colours';
+import {
+  GEL_GREAT_PRIMER,
```

Comment:
Hey, soz just spotted this. Can we update these to use the new functions, and receive a script prop please? An example can be found here https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/src/index.jsx#L18